infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -21,12 +20,12 @@ export class AtatWebApiStack extends cdk.Stack {
     this.templateOptions.description = "Resources to support the ATAT application API";
 
     // Create a shared DynamoDB table that will be used by all the functions in the project.
-    const table = new dynamodb.Table(this, "AtatTable", {
-      partitionKey: { name: "id", type: dynamodb.AttributeType.STRING },
-      billingMode: dynamodb.BillingMode.PROVISIONED,
-      readCapacity: 1,
-      writeCapacity: 1,
-      removalPolicy: props?.removalPolicy,
+    const { table } = new SecureTable(this, "AtatTable", {
+      tableProps: {
+        partitionKey: { name: "id", type: dynamodb.AttributeType.STRING },
+        billingMode: dynamodb.BillingMode.PAY_PER_REQUEST,
```

Comment:
Just a note to others, per the CloudFormation docs, changing the billing mode requires "No Interruption". We're about to test that in Dev :)